common/src/main/java/com/revenuecat/purchases/common/networking/Endpoint.kt

```diff
@@ -33,6 +33,22 @@ sealed class Endpoint(val pathTemplate: String, val name: String) {
     }
 
     val supportsSignatureValidation: Boolean
```

Comment:
I realized this was also outdated in iOS:
```suggestion
    val supportsSignatureVerification: Boolean
```